src/coreclr/vm/jitinterface.cpp

```diff
@@ -7593,9 +7593,8 @@ static void getMethodInfoHelper(
                              (ftn->RequiresInstMethodTableArg() ? CORINFO_GENERICS_CTXT_FROM_METHODTABLE : 0) |
                              (ftn->RequiresInstMethodDescArg() ? CORINFO_GENERICS_CTXT_FROM_METHODDESC : 0)));
 
-    // EEJitManager::ResolveEHClause and CrawlFrame::GetExactGenericInstantiations
-    // need to be able to get to CORINFO_GENERICS_CTXT_MASK if there are any
-    // catch clauses like "try {} catch(MyException<T> e) {}".
+    // EEJitManager::ResolveEHClause needs to be able to get to CORINFO_GENERICS_CTXT_MASK
+    // if there are any catch clauses like "try {} catch(MyException<T> e) {}".
     // Such constructs are rare, and having to extend the lifetime of variable
     // for such cases is reasonable
```

Comment:
```suggestion
```
This comment is not correct. `EEJitManager::ResolveEHClause` does not need to get to `CORINFO_GENERICS_CTXT_MASK`. For reference, this need was removed by #72721.